presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/MetastoreOperationStats.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.hive.metastore;
+
+import org.joda.time.DateTime;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static java.util.Objects.requireNonNull;
+
+public class MetastoreOperationStats
+{
+    private final MetastoreOperationType operationType;
+    private final List<DateTime> lastDataCommitTimes;
+    private static final int DUMMY_MILLISECONDS = 0;
```

Comment:
why do we need this; can we just create an empty list?

Reply:
Do you mean that we don't need this class?

Reply:
Just use a list of integers instead of using a class to wrap them?

Reply:
meaning why do we need `DUMMY_MILLISECONDS`?

Reply:
The normal purpose for using constants is to 1. remove the duplicates; 2. add meaning to a magic number. Here the purpose is 2.

Reply:
Nvm, I got your idea. We should not create this variable or constant at all.